tests/fixtures/make-registration-form/tests/it_generates_registration_with_entity_and_authenticator.php

```diff
@@ -54,7 +54,7 @@ public function testRegistrationValidationError()
         $form['registration_form[plainPassword]'] = 'foo';
         $client->submit($form);
 
-        $this->assertSame(200, $client->getResponse()->getStatusCode());
+        $this->assertSame(422, $client->getResponse()->getStatusCode());
```

Comment:
This smells funny - there should have been a test failure before _this_ PR to show that `422` was the expected status code. Makes me think there is something up with the test suite... 

Can you push a commit that changes this back to `200` & then push _another_ commit to change this back to `422` so we can see the tests fail in CI? Thank you in advance

Reply:
It's actually quite strange, I myself didn't understand how the tests could work with a status code of 200 in the case of an invalid form.

But I didn't notice straight away that the status of the code was wrong when I pushed my first commit, it's thanks to the CI tests that failed and told me that it was expecting a code with status 422 in the event of an invalid form.

Reply:
I've been chewing on this all morning and _think_ the reason why we didn't see this fail before your PR has to do with an exception that was caught in the test but didn't bubble up. MakerBundles test setup is pretty complex. when we run `vendor/bin/simple-phpunit` each `Make*Test` actually creates a new "project" -> runs the maker -> runs tests within the project -> reports back to the original `Make*Test`... sometimes things get lost in translation... Anyhow, thats out of scope for this PR.. :)